### ISSUE TITLE
Android build fails

### ISSUE THREAD

**Issue by igorskh** [malyok1627/recipe_mgmt_app#2]
When running the following command:
```bash
flutter build appbundle --target-platform android-arm,android-arm64
```
The error occurs:
```text
[!] Your app isn't using AndroidX.
    To avoid potential build failures, you can quickly migrate your app by following the steps on
    https://goo.gl/CP92wY.
recipe_mgmt_app/recipe_mgmt_app/android/app/src/main/AndroidManifest.xml:22:18-86 Error: Attribute application@appComponentFactory value=(androidx.core.app.CoreComponentFactory) from [androidx.core:core:1.0.0] AndroidManifest.xml:22:18-86 is also present at [com.android.support:support-compat:28.0.0] AndroidManifest.xml:22:18-91 value=(android.support.v4.app.CoreComponentFactory). Suggestion: add 'tools:replace="android:appComponentFactory"' to <application> element at AndroidManifest.xml:15:5-38:19 to override. 
FAILURE: Build failed with an exception.* What went wrong: Execution failed for task ':app:processReleaseManifest'.> Manifest merger failed : Attribute application@appComponentFactory value=(androidx.core.app.CoreComponentFactory) from [androidx.core:core:1.0.0] AndroidManifest.xml:22:18-86   is also present at [com.android.support:support-compat:28.0.0] AndroidManifest.xml:22:18-91 value=(android.support.v4.app.CoreComponentFactory).
Suggestion: add 'tools:replace="android:appComponentFactory"' to <application> element at AndroidManifest.xml:15:5-38:19 to override.

Running Gradle task 'bundleRelease'...                                                                        BUILD FAILED in 7m 36s            
AndroidX incompatibilities may have caused this build to fail. Please migrate your app to AndroidX. See
https://goo.gl/CP92wY.
Gradle task bundleRelease failed with exit code 1
```

Also while running on the emulator:
```text
Launching lib/main.dart on ONEPLUS A3010 in debug mode...
[!] Your app isn't using AndroidX.
    To avoid potential build failures, you can quickly migrate your app by following the steps on https://goo.gl/CP92wY.
/Users/igorkim/dev/recipe_mgmt_app/recipe_mgmt_app/android/app/src/main/AndroidManifest.xml:22:18-91 Error:
	Attribute application@appComponentFactory value=(android.support.v4.app.CoreComponentFactory) from [com.android.support:support-compat:28.0.0] AndroidManifest.xml:22:18-91
	is also present at [androidx.core:core:1.0.0] AndroidManifest.xml:22:18-86 value=(androidx.core.app.CoreComponentFactory).
	Suggestion: add 'tools:replace="android:appComponentFactory"' to <application> element at AndroidManifest.xml:15:5-38:19 to override.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute application@appComponentFactory value=(android.support.v4.app.CoreComponentFactory) from [com.android.support:support-compat:28.0.0] AndroidManifest.xml:22:18-91
  	is also present at [androidx.core:core:1.0.0] AndroidManifest.xml:22:18-86 value=(androidx.core.app.CoreComponentFactory).
  	Suggestion: add 'tools:replace="android:appComponentFactory"' to <application> element at AndroidManifest.xml:15:5-38:19 to override.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4s
AndroidX incompatibilities may have caused this build to fail. Please migrate your app to AndroidX. See https://goo.gl/CP92wY.
Gradle task assembleDebug failed with exit code 1
Exited (sigterm)
```

Maybe migrating would help? 
```
To avoid potential build failures, you can quickly migrate your app by following the steps on
    https://goo.gl/CP92wY.
```

**Comment by igorskh** [comment]
#1 migrates to Android X and adds GitHub actions for building

**Comment by malyok1627** [comment]
fixed in f18a63d70e2b36363966167ae66feefabd11ecb2
